docs/sources/panels/visualizations/_index.md

```diff
@@ -1,12 +1,10 @@
 +++
-title = "Visualizations"
+title = "Visualization options"
```

Comment:
@oddlittlebird 
Any reason for his change to the menu category? this makes this category sound like it contains options, just makes it unnecessarily long, would it not be more clear if the navigation was just "Visualizations" (like it is in current latest docs), the panels or data sources categories are not "Data source options" etc.